blazor/dashboard-layout/enablePersistence.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: EnablePersistence in Blazor Dashboard Layout Component | Syncfusion

```

Comment:
change tittle to **State Persistence** instead of EnablePersistence.

Reply:
Done.